packages/runtime/container-runtime/tsconfig.json

```diff
@@ -8,7 +8,8 @@
         "rootDir": "./src",
         "outDir": "./dist",
         "types": [
-            "mocha"
+            "mocha",
+            "node"
```

Comment:
Needed to run with "Debug current test" profile

Reply:
I don't think we want to do this, since container-runtime package runs in browser. It doesn't actually hurt anything, but it makes it easier to accidentally use node-specific types.

Is there a way to only include the node-specific typings that are needed for the test in the test?
Even if it just means stubbing them locally?
FYI - @curtisman , @heliocliu

Reply:
You can also add a separate tsconfig file that extends the regular one and adds the types like this just for test; common-utils does this for its tests

Reply:
Ah very interesting.  I will revert, looks like we need to do some cleanup across the repo here, to get a tsconfig in the test dir everywhere so the launch config can use a consistent path to the test tsconfig.

Reply:
Opened #4397 for this.